src/command/check/check.ts

```diff
@@ -144,13 +144,17 @@ async function checkInstall(services: RenderServices) {
   info(`      Version: ${quartoConfig.version()}`);
   if (quartoConfig.version() === "99.9.9") {
     // if they're running a dev version, we assume git is installed
+    // and QUARTO_ROOT is set to the root of the quarto-cli repo
     // print the output of git rev-parse HEAD
-    const gitHead = await execProcess({
-      cmd: ["git", "rev-parse", "HEAD"],
-      stdout: "piped",
-    });
-    if (gitHead.stdout) {
-      info(`      commit: ${gitHead.stdout.trim()}`);
+    if (Deno.env.get("QUARTO_ROOT")) {
```

Comment:
In other places we do `join(quartoConfig.sharePath(), "../..")` and do not assume `QUARTO_ROOT` is set. 

Though it seems better to assume it is, at least it should from interactive run of `quarto check` from a dev version

Reply:
> And it seems I should do the same here, because `QUARTO_ROOT` is not set in our every run !

Do we understand why QUARTO_ROOT is not set?

Reply:
So my recent test seems to confirm this is not set in CI.  I need to understand why... Because it seems we do set the variable and export 🤔 

https://github.com/quarto-dev/quarto-cli/blob/30e0873eb211068c3e2672abe822ce1803299362/package/scripts/common/quarto#L24-L33

https://github.com/quarto-dev/quarto-cli/blob/30e0873eb211068c3e2672abe822ce1803299362/package/scripts/windows/quarto.cmd#L9

I am probably missing something 🤷‍♂️

Reply:
No I am wrong. This works ! 

It was really a Typescript checking issue. Now storing in a constant and checking it makes it passe ok. 
And quarto check shows the commit on CI. 

Sorry for the noise. It was not an issue .